### PR TITLE
Feature/orgs consent ui fixes

### PR DESCRIPTION
**Pull request by achen2401**
address additional requirements for this story:  https://www.pivotaltracker.com/story/show/143463375

- set default consent if no consent agreement is available for parent organization
- verbiage changes
- add fix to remove existing consent(s) from un-selected clinic(s)
- re-factor code to allow reference from common template (e.g. consent agreement template)

### CODE REVIEW

File: portal/static/js/main.js [hunk 1]
```diff
@@ -714,7 +713,8 @@ var fillContent = {
                     var expiresDate = convertUserDateTimeByLocaleTimeZone(item.expires, userTimeZone, userLocale);
                     var editorUrlEl = $("#" + orgId + "_editor_url");
                     var isFake = (/fake/.test(item.agreement_url));
```
Comment (pbugni) [inline portal/static/js/main.js]: I believe we've not killed off the `fake` link.  Is this test still necessary?

File: portal/static/js/main.js [hunk 2]
```diff
@@ -728,7 +728,7 @@ var fillContent = {
                                     sDisplay = "<span class='text-success small-text'>Consented / Enrolled</span>";
                                     cflag = "consented";
                             } else if (se && ir && !sr) {
-                                    sDisplay = "<span class='text-warning small-text'>Suspend Data Collection and Reports</span>";
+                                    sDisplay = "<span class='text-warning small-text'>Suspend Data Collection and Report Historic Data</span>";
```
Comment (pbugni) [inline portal/static/js/main.js]: Needs the gettext wrapper.

File: portal/static/js/main.js [hunk 3]
```diff
@@ -771,12 +771,20 @@ var fillContent = {
...
+                            content: sDisplay + (editable && consentStatus == "active"? '&nbsp;&nbsp;<a data-toggle="modal" data-target="#consent' + index + 'Modal" ><span class="glyphicon glyphicon-pencil" aria-hidden="true" style="cursor:pointer; color: #000"></span></a>' + modalContent: ""),
                             "_class": "indent"
                         },
                         {
-                            content: (isFake? "--" : "<span class='agreement'><a href='" + item.agreement_url + "' target='_blank'><em>View</em></a></span>" +
-                            ((editorUrlEl.length > 0 && hasValue(editorUrlEl.val())) ? ("<div class='button--LR' " + (editorUrlEl.attr("data-show") == "true" ?"data-show='true'": "data-show='false'") + "><a href='" + editorUrlEl.val() + "' target='_blank'>Edit in Liferay</a></div>") : ""))
+                            content: function(item) {
+                                var s = "";
+                                if (isFake) s = "--";
```
Comment (pbugni) [inline portal/static/js/main.js]: More fake tests that i think we can now eliminate.

File: portal/static/js/main.js (hunk 2 above)
Comment (achen2401) [inline portal/static/js/main.js]: Paul, not sure if gettext can be applied in this context as the text is generated dynamically from client side.

**Pull request merged**